lib/validator.ex

```diff
@@ -0,0 +1,176 @@
+defmodule PhoenixSwagger.Validator do
+
+  @moduledoc """
+  The PhoenixSwagger.Validator module provides converter of
+  swagger schema to ex_json_schema structure for further validation.
+
+  There are two main functions:
+
+    * parse_swagger_schema/1
+    * validate/2
+
+  Before `validate/2` will be called, a swagger schema should be parsed
+  for futher validation with the `parse_swagger_schema/1`. This function
+  takes path to a swagger schema and returns it in ex_json_schema format.
+
+  During execution of the `parse_swagger_schema/1` function, it creates
+  the `validator_table` ets table and stores associative key/value there.
+  Where `key` is an API path of a resource and `value` is input parameters
+  of a resource.
+
+  To validate of a parsed swagger schema, the `validate/1` should be used.
+
+  For more information, see more in ./phoenix_swagger/tests/ directory.
+  """
+
+  @doc """
+  The `parse_swagger_schema/1` takes path to a swagger schema, parses it
+  into ex_json_schema format and store to the `validator_table` ets
+  table.
+
+  Usage:
+
+      iex(1)> parse_swagger_schema("my_json_spec.json")
+      [{"/person",  %{'__struct__' => 'Elixir.ExJsonSchema.Schema.Root',
+                      location => root,
+                      refs => %{},
+                      schema => %{
+                        "properties" => %{
+                          "name" => %{<<"type">> => <<"string">>},
+                          "age" => %{<<"type">> => <<"integer">>}
+                        }
+                      }
+                    }
+      }]
+
+  """
+  def parse_swagger_schema(spec) do
+    table = case :ets.info(:validator_table) do
+              :undefined ->
+                :ets.new(:validator_table, [:public,:named_table])
+                :validator_table
```

Comment:
@surik Did you mean elixir attribute? like `@table :validator_table` or somethine else?